Assets/BossRoom/Scripts/Client/Game/Action/DashAttackActionFX.cs

```diff
@@ -9,6 +9,13 @@ namespace BossRoom.Visual
     /// But on the client, we visualize this as the character dashing across the screen. The dashing begins after
     /// ExecTimeSeconds have elapsed.
     /// </summary>
+    /// <remarks>
+    /// This script moves the character visualization during the "dash", which is normally performed by
+    /// <see cref="ClientCharacterVisualization"/>. If both scripts try to move the visualization at the
+    /// same time it would result in unpredictable movement stuttering, but it's fine in this case because
+    /// the character has been stationary for ExecTimeSeconds before we start moving it (so the regular
+    /// visualization code will be dormant.)
```

Comment:
These last two lines leave me a little confused. Shouldn't there be a clash? `ClientCharacterVisualization` should be applying SmoothMove towards the ServerCharacter position on every Update() right? So is it really dormant?

Reply:
Hmm, I think you're right -- the `ClientCharacterVisualization` will continue to pull the visualization back toward the server character position. But the movement isn't strong enough to be noticeable due to the very short time we spend moving the visualization.

Should I add the visualization-disabling boolean back? Or just adjust the comment?

Reply:
That's a good question. Since the movement workaround removal was suggested by @SamuelBellomo, let's get his opinion here. It doesn't feel right to have two concurrent movement behaviours, however.

Reply:
SmoothMove is called before ActionFX.Update() in ClientCharacterVisualisation
It's normal this lerp here overrides the interpolation. (let's make sure we update our comment to reflect this)

Reply:
I added a suggestion change. Once we take this in, I'm good